src/Aspire.ProjectTemplates/templates/aspire-apphost/.template.config/template.json

```diff
@@ -6,36 +6,88 @@
     ".NET Aspire",
     "Cloud"
   ],
-  "name": ".NET Aspire 9 App Host",
+  "name": ".NET Aspire App Host",
   "defaultName": "AppHost",
-  "description": "A project template for creating a .NET Aspire 9 app host (orchestrator) project.",
-  "shortName": ["aspire-apphost", "aspire-apphost-9"],
+  "description": "A project template for creating a .NET Aspire app host (orchestrator) project.",
+  "shortName": "aspire-apphost",
   "sourceName": "Aspire.AppHost1",
```

Comment:
Also, missing ide folder and icon for the AppHost template, as well as ide.host.json - note that you'll need to push in the AspireVersion symbol and mark as isVIsible as well.